### PR TITLE
Psg 4891 Use repo templates in passage-flutter

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
# Pull Request Template

## What's New?

Uses repo templates to format README, LICENSE, and remove community files in favour of using organization community files.

## Screenshots (if appropriate):

(Note Passage logo will adhere to system theme, screenshot taken from vscode preview)
CleanShot 2024-10-18 at 12 08 44@2x

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have manually tested my code thoroughly
- [ ] I have added/updated inline documentation for public facing interfaces if relevant
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing integration and unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Jira Ticket

Please add the Jira ticket number and link.

- [ ] Jira Ticket: [PSG-XXXX](https://passage-identity.atlassian.net/browse/PSG-XXXX)

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,77 @@
...
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex**             | ✖️ For Passkey Flex, check out the [Passkey Flex APIs](https://docs.passage.id/flex/apis)                                                                                                                                 |
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✅                                                                                                                                                                                                                        |
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready**          | ✖️ For Passkey Ready, check out [Authentikit for Android](https://github.com/passageidentity/passage-android/tree/main/authentikit) and [Authentikit for iOS](https://github.com/passageidentity/passage-authentikit-ios) |
```
Comment (jenn-macfarlane) [inline README.md:23]: Do we want to link to mobile-specific versions of Authentikit here?

Comment (rickycpadilla) [reply README.md:23]: Looks great!

**Review by rickycpadilla — APPROVED** [review]

**Comment by agebhardt** [comment]
@jenn-macfarlane there is a Passage logo that works in the template. Can that not be used here for some reason?

**Comment by jenn-macfarlane** [comment]
@rickycpadilla any idea why these integration tests for android would be failing? I can't think of why my changes would cause this 🤔 

CleanShot 2024-10-18 at 12 49 24@2x

**Comment by jenn-macfarlane** [comment]
Waiting to merge until the [Android](https://passage-identity.atlassian.net/browse/PSG-4895) ticket is done. Reason being this way I will have a README for Android/authentikit to link to in the products comparison table instead of just the folder in github.

**Pull request merged**